bindings_node/src/mls_client.rs

```diff
@@ -3,21 +3,28 @@ use std::ops::Deref;
 use std::sync::Arc;
 
 use crate::conversations::NapiConversations;
-use napi::bindgen_prelude::{Error, Result, Uint8Array};
+use napi::bindgen_prelude::{BigInt, Error, Result, Uint8Array};
 use napi_derive::napi;
 use xmtp_api_grpc::grpc_api_helper::Client as TonicApiClient;
 use xmtp_cryptography::signature::ed25519_public_key_to_address;
-use xmtp_id::associations::{Erc1271Signature, RecoverableEcdsaSignature};
+use xmtp_id::associations::{AccountId, Erc1271Signature, RecoverableEcdsaSignature, Signature};
 use xmtp_mls::builder::ClientBuilder;
 use xmtp_mls::identity::IdentityStrategy;
 use xmtp_mls::storage::{EncryptedMessageStore, EncryptionKey, StorageOption};
 use xmtp_mls::Client as MlsClient;
 
 pub type RustXmtpClient = MlsClient<TonicApiClient>;
 
+#[derive(Eq, Hash, PartialEq)]
+enum SignatureType {
+  RecoverableEcdsaSignature,
+  Erc1271Signature,
+}
+
 #[napi]
 pub struct NapiClient {
   inner_client: Arc<RustXmtpClient>,
+  signatures: HashMap<SignatureType, Box<dyn Signature>>,
```

Comment:
Like we discussed, the key of type should be `MemberIdentifier` instead of `SignatureType`.

Reply:
updated: [Update signatures key type](https://github.com/xmtp/libxmtp/pull/800/commits/a9e841a2b238c5cef40d3e3a292966c186fe1570)